pages/page.py

```diff
@@ -11,11 +11,10 @@
 import time
 
 import requests
-from unittestzero import Assert
+from requests.exceptions import Timeout
```

Comment:
Nit: If you're going to move the import for `requests` in order to alphabetize (I assume that's why), perhaps you should put the `selenium` imports into alphabetical order as well.